package.json

```diff
@@ -10,8 +10,7 @@
     "upgrade-sdk:next": "npm install --save-dev @dcl/sdk@next"
   },
   "devDependencies": {
-    "@dcl/asset-packs": "latest",
```

Comment:
If we remove this from the dependencies, the Editor will disable the smart items from the inspector's catalog: https://github.com/decentraland/editor-sdk7/blob/main/src/views/inspector/webview.ts#L45

Reply:
yes, the idea is to have another template for the editor that uses smart-items. 
So if you are creating a project without them you dont install unnecessary dependencies.